front_end/html/edit_course.html

```diff
@@ -65,3 +65,28 @@ <h2>Create course</h2>
 </p>
 
 </form>
+
+<script>
+  $('div[contenteditable]').bind('paste', function(e) {
```

Comment:
This same code is used in multiple locations. I forget where it is, but I believe we have a file in CodeBuddy that includes Javascript functions that are shared across all pages. Please find a way to move the code into a function and put it in that file and then invoke the function from here.

Reply:
I found the shared.js file and put the function in there. Then I linked shared.js at the bottom of each of the three edit files. I wasn't sure if it would be better to link it in all 3 or only in footer.html so that it'd be linked in one place or if that would make performance costs from every page being able to access functions they don't need.

Reply:
I would say it's better to link it from the specific pages where it's used than from footer.html for the reasons you mentioned.